docs/server-side/v2/Destinations/recording.mdx

```diff
@@ -62,27 +62,27 @@ Another way to enable SFU recording for a room is to choose `Enabled` for record
 By default recordings will be uploaded to 100ms storage and a pre-signed URL for the same will be provided to customers via a webhook. The pre-signed URL will expire in 12 hours.
 Customers can also configure the recordings to be stored in their cloud storage. Following are the configurations for the same.
 
-| Name        | Type     | Description                                                                                           | Required |
-| :---------- | :------- | :---------------------------------------------------------------------------------------------------- | :------- |
-| type        | `string` | Upload Destination type. Currently, only `s3` is supported                                            | Yes      |
-| location    | `string` | Name of the AWS s3 bucket in which you want to store all recordings                                   | Yes      |
-| prefix      | `string` | Upload prefix path                                                                                    | No       |
-| options     | `object` | Additional configurations of type `Options` to be used for uploading                                  | No       |
-| credentials | `object` | Object of type `Credentials`. This is used to share AWS credentials to access the s3 bucket specified | No       |
+| Name        | Type     | Description                                                                                                       | Required |
+| :---------- | :------- | :---------------------------------------------------------------------------------------------------------------- | :------- |
+| type        | `string` | Upload destination type. Currently, `s3` (AWS), `gs` (Google Cloud Storage), `oss` (Alibaba Cloud) are supported. | Yes      |
```

Comment:
@chdinesh1089 should oss be in all caps?
[Reference](https://www.alibabacloud.com/product/object-storage-service)

Reply:
In the parenthesis? (In the code block it has to be lower case as the api supports only that.)
In the parenthesis I've mentioned it 'Alibaba Cloud' thinking OSS isn't used commonly and also easier to find if someone just searches alibaba in the docs.

Should we change it?

Reply:
got it, my bad.